src/components/Event/Event.st.css

```diff
@@ -0,0 +1,130 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback, font;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+/*Defaults*/
+:vars {
+    DefaultFontStyle: "{theme: 'Body-M', size: '14px', lineHeight: '1em'}";
+    DefaultTimeColor: color-4;
+    DefaultTitleColor: color-5;
+    DefaultAllDayBackground: color-8;
+    DefaultAllDayOpacity: 0.2; 
+    DefaultTopPadding: 4px;
+    DefaultRightPadding: 4px;
+    DefaultBottomPadding: 4px;
+    DefaultLeftPadding: 12px;
+    BeforeColor: color(color-4);
+    BeforeOpacity: 0.06;
+}
+
+/*Overrides*/
+:vars {
+    FontStyle: --overridable;
+    TimeColor: --overridable;
+    TitleColor: --overridable;
+    AllDayBackground:  --overridable;
+    AllDayOpacity: --overridable;
+    TopPadding: --overridable;
+    RightPadding: --overridable;
+    BottomPadding: --overridable;
+    LeftPadding: --overridable;
```

Comment:
Okay, so after consulting with our designers, the overridable variables should be like in this table https://zeroheight.com/7sjjzhgo2/p/48969b-calendar/t/34d1d4 :

* textFont
* timeColor
* titleColor
* bgColor
* hoverOverlayColor
* selectedOverlayColor
* fulldayBgColor

No padding is needed here, it should be fixed in the component